src/test/librbd/object_map/test_mock_SnapshotRemoveRequest.cc

```diff
@@ -268,18 +268,27 @@ TEST_F(TestMockObjectMapSnapshotRemoveRequest, ScrubCleanObjects) {
 
   librbd::ImageCtx *ictx;
   ASSERT_EQ(0, open_image(m_image_name, &ictx));
+  librbd::NoOpProgressContext prog_ctx;
+  uint64_t size = 4294967296; // 4GB = 1024 * 4MB
+  ASSERT_EQ(0, resize(ictx, size));
+  
+  // update image objectmap for snap inherit 
+  ceph::BitVector<2> object_map;
+  object_map.resize(1024);
```

Comment:
I'm not sure what this is attempting to test. The object map is tiny so you won't see any performance issues (and the perf issues really only occurred when loading the object map from disk in 4K chunks since that caused the indexing to switch from O(1) to O(n)).

If the goal is to test that object map functions with very large images, the best place to do that would be within an integration test against a live cluster.

Reply:
i aggree with you about what this test is attempting.
as object_map.resize(1024) lied there previously, which is just moved to here by me.